Translated/Story/Side Manon/ssc_502060.csv

```diff
@@ -0,0 +1,36 @@
+ssc_502060_0010#0,"""Hmmm...<br>This signal..."""
+ssc_502060_0020#0,"""As a theoretical value,<br>it'd be around... this much?"""
+ssc_502060_0030#0,"""...Oh my.<br>Where did you come from?"""
+ssc_502060_0040#0,"""You startled me again.<br>You're very good at hiding your presence."""
+ssc_502060_0050#0,"""What are you up to?"""
+ssc_502060_0060#0,"""Taking an interest in those devices?"""
+ssc_502060_0070#0,"""I've been studying these<br>Photon Tuners recently."""
+ssc_502060_0080#0,"""With permission from the<br>people of Retem, of course."""
+ssc_502060_0090#0,"""I've been taking data<br>on these Photon Tuners."""
+ssc_502060_0100#0,"""I did make sure to ask Mura<br>in advance, so don't worry."""
+ssc_502060_0110#0,"""However, what I'm really interested in<br>is not the Tuners themselves..."""
+ssc_502060_0120#0,"""But rather that which they are used to broadcast:<br>Nadereh's \""Power of Song\""."""
+ssc_502060_0130#0,"""Her power is closely related to<br>both her inner and outer Photons."""
+ssc_502060_0140#0,"""Which means that, in theory, it is<br>something similar to a Technique."""
+ssc_502060_0150#0,"""However, a Technique that affects the whole desert should<br>require an equally enormous amount of Photons."""
+ssc_502060_0160#0,"""Even the strongest Force or Techter would have extreme<br>difficulty casting such a Technique by themselves."""
+ssc_502060_0170#0,"""So I thought to start by<br>investigating these devices..."""
+ssc_502060_0180#0,"""Sadly, however, I've<br>reached a dead end."""
+ssc_502060_0190#0,"""From what I've gathered thus far, the Tuner amplification<br>isn't strong enough to explain her power."""
+ssc_502060_0200#0,"""Which leads me to believe that most of the power<br>comes entirely from Nadereh herself."""
+ssc_502060_0210#0,"""If this is the case, then rather than as a Technique,<br>it could be forces operating on a more fundamental level..."""
+ssc_502060_0220#0,"""By some chance, this could be \""Magic\""...<br>or something similar to it."""
+ssc_502060_0230#0,"""Magic?"""
+ssc_502060_0240#0,"""Have you tried asking Nadereh?"""
+ssc_502060_0250#0,"""... No, sorry, ignore that.<br>I got caught up in speculation."""
+ssc_502060_0260#0,"""I'll have to investigate just a bit more."""
+ssc_502060_0270#0,"""That's... true, but I doubt even she herself<br>would have a theory as to how it works."""
```

Comment:
This line follows on weirdly from what I think is the player dialogue it's responding to. "Have you tried asking Nadereh?" "That's... true".

Reply:
fixt